eureka/6b09030e95e118a784ca7ec616398a4f0e384bcd/README.md

```diff
@@ -4,31 +4,43 @@
 ## 2. Checkout to merge the commit hash
     git checkout 6b09030e95e118a784ca7ec616398a4f0e384bcd
 
-## 3. Edit the local.properties add the location sdk. Example:
```

Comment:
Is this step no longer required? Did you apply some local configuration regarding the sdk file?

Reply:
I replied it from your fork. I generated the transformed jar successfully, and this step was not required.